packages/ingest/test/test-S3ProviderClient.js

```diff
@@ -68,7 +68,22 @@ test.serial('S3ProviderClient.download downloads a file to local disk', async (t
   const localPath = './tmp.json';
   t.teardown(() => fs.unlinkSync(localPath));
 
-  await s3ProviderClient.download(t.context.sourceKey, localPath);
+  await s3ProviderClient.download({ remotePath: t.context.sourceKey, localPath });
+  t.true(fs.existsSync(localPath));
+  t.is(fs.readFileSync(localPath).toString(), t.context.fileContent);
+});
+
+test.serial('S3ProviderClient.download downloads a file with source_bucket set to local disk', async (t) => {
```

Comment:
This should probably be 'S3ProviderClient.download downloads a file  to local disk when source_bucket is set '

Reply:
006bb605b